security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -130,6 +136,13 @@ type SecretController struct {
 	// Used to coordinate with label and check if this instance of Citadel should create secret
 	istioCaStorageNamespace string
 	monitoring              monitoringMetrics
+
+	// Extra trust anchors related state
+	extraTrustAnchorsController cache.Controller
+	extraTrustAnchorMu          sync.Mutex
+	extraTrustAnchors           map[string]map[string]string // resource name -> field name -> field value
```

Comment:
Consider adding comments with examples for extraTrustAnchors, e.g., explaining 'resource name', 'field name', 'field value', what exactly they are and why they are needed, and etc. 
Similarly,  consider adding comments for primaryTrustAnchor and combinedTrustAnchorsBundle.